bool position_state

---
bool response

